R/consense.R

```diff
@@ -0,0 +1,53 @@
+
+consense <- function(sN,minEdgeCC=length(sN),noLog = FALSE,silent = FALSE) {
+
+    #'Find consensus subnetworks
+    #'
+    #   sN: List of subnetworks resulting after edge removal and
+    #'@param sN List of subnetworks
+    #'@param minEdgeCC minimum number of subnetworks an edge appears in to be kept
+    #'@param noLog if FALSE, save process to an .Rdata log file
+    #'@param silent if FALSE, print process to console in real time
+    #'@return A list of consensus subnetworks as igraph objects
+
+    #INPUTS:
+    #   sN: List of subnetworks
+    #   minEdgeCC: minimum number of subnetworks an edge appears in to be kept#   method: method of edge removal
+    #   noLog: if FALSE, save process to an .Rdata log file#   delta: threshold for edge removal for given method
+    #   silent: if FALSE, print process to console in real time#   EGG: 'Heat Diffusion' annotated AGG, iGraph object.
+
+    #OUTPUT:
+    #   cSN: List of subnetworks resulting after edge removal and
+    #       subnetwork construction
+
+
+    #minEdgeCC = 0 #length(outputGraphs)
+    numGraphs <- length(outputGraphs)
+    sNEdges <- vector(mode="list", length=numGraphs)
+    sNVerts <- vector(mode="list", length=numGraphs)
+    edgePairsList <- c() #vector(mode="list", length=numGraphs)
+    for (i in 1:numGraphs){
```

Comment:
Thanks Boris. I was trying to create the pasted combined vertices in `edgePairs` on line 40. I was using a list, and tried to create the count table from that list, but I agree that your strategy makes more sense.